platform/android/src/style/statement.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <mbgl/style/filter.hpp>
+#include <mbgl/util/noncopyable.hpp>
+
+#include <jni/jni.hpp>
+
+namespace mbgl {
+namespace android {
+
+class Statement : private mbgl::util::noncopyable {
```

Comment:
Please wrap this in `Filter`, so it mirrors the Java class hierarchy and reads better